gwak/deploy/deploy.smk

```diff
@@ -28,7 +28,7 @@ rule infer:
     output:
         artefact = directory('output/infer/{deploymodels}')
     shell:
-        'set -x; cd deploy; CUDA_VISIBLE_DEVICES=1 poetry run python \
+        'set -x; cd deploy; CUDA_VISIBLE_DEVICES=0 poetry run python \
```

Comment:
Is it possible to switch to environments instead of poetry? it is easy with Snakemake, here is an example 
```
rule some_rule:
    input:
        "data/input.txt"
    output:
        "results/output.txt"
    conda:
        "envs/env1.yaml"
    shell:
        "some_command --input {input} --output {output}"
```
and then run with `snakemake --use-conda`.
So, if a special env is needed, we can just define a separate `yaml` for it and that's ensure there is no conflict. It looks like Eric had a lot of issues running inference due to collision between conda and poetry